server/test/services/question/types/QuestionDefinitionTest.java

```diff
@@ -685,6 +689,7 @@ public void validate_throwsExceptionWhenQuestionTypesMismatched() {
             + " updated version.");
   }
 
+  @SuppressWarnings("unused") // Used by @Parameters
```

Comment:
I was lazy about the comment; updated. I get unused warnings in my IDE. I guess that's not one that we turn into an error though.

SuppressWarnings is used sporadically around the codebase but obviously not enforced so I guess it doesn't really matter either way. I prefer to have it (or more importantly the comment) to explain what the method is for since the name isn't all that illustrative.

Reply:
😄